app/controllers/admin/backups_controller.rb

```diff
@@ -8,7 +8,20 @@ class Admin::BackupsController < Admin::AdminController
 
   before_action :ensure_backups_enabled
   skip_before_action :check_xhr, only: %i[index show logs check_backup_chunk upload_backup_chunk]
-  skip_before_action :ensure_backups_enabled, only: %w[show status index email]
+  skip_before_action :ensure_backups_enabled, only: %i[show status index email]
+
+  allow_in_readonly_mode :create,
```

Comment:
Explicitly listed all the "POST/PUT/DELETE" actions that an admin can trigger when dealing with backups while in "readonly" mode.